shared/profile/index.js

```diff
@@ -1,14 +1,17 @@
 // @flow
+import EditProfile from './edit-profile'
 import React, {Component} from 'react'
-import {connect} from 'react-redux'
 import Render from './render'
-import EditProfile from './edit-profile'
-import type {Props} from './render'
 import flags from '../util/feature-flags'
+import type {MissingProof} from '../common-adapters/user-proofs'
+import type {Props} from './render'
+import {addProof} from '../actions/profile'
+import {connect} from 'react-redux'
 import {getProfile, updateTrackers} from '../actions/tracker'
-import {routeAppend, navigateUp} from '../actions/router'
-import {openInKBFS} from '../actions/kbfs'
 import {isLoading} from '../constants/tracker'
+import {openInKBFS} from '../actions/kbfs'
+import {routeAppend, navigateUp} from '../actions/router'
+import ProveEnterUsername from './prove-enter-username-container'
 
 class Profile extends Component<void, Props, void> {
   static parseRoute (currentPath, uri) {
```

Comment:
originally i didn't need this component but i do for the parseRoute function. when we revisit the meta-navigator we could remove this requirement then we could get rid of this wrapper